modules/outbound-command-event-handler/test/integration/sample_events/sdk_outbound_bulk_accept_party_info_received.test.ts

```diff
@@ -30,12 +30,18 @@ import { SDKSchemeAdapter } from "@mojaloop/api-snippets";
 import { DefaultLogger } from "@mojaloop/logging-bc-client-lib";
 import { ILogger } from "@mojaloop/logging-bc-public-types-lib";
 
-import { SDKOutboundBulkAcceptPartyInfoReceivedDmEvt, ISDKOutboundBulkAcceptPartyInfoReceivedDmEvtData } from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
-import { KafkaDomainEventProducer, IKafkaEventProducerOptions } from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
+import {
+  KafkaDomainEventProducer,
+  IKafkaEventProducerOptions,
+  RedisBulkTransactionStateRepo,
+  IRedisBulkTransactionStateRepoOptions,
+  IndividualTransferInternalState,
+  SDKOutboundBulkAcceptPartyInfoReceivedDmEvt,
+  ISDKOutboundBulkAcceptPartyInfoReceivedDmEvtData
+} from '@mojaloop/sdk-scheme-adapter-private-shared-lib'
 import { randomUUID } from "crypto";
 
-import { BulkTransactionAgg, IndividualTransferEntity, IndividualTransferInternalState, IndividualTransferState } from '../../../src/domain'
-import { RedisBulkTransactionStateRepo, IRedisBulkTransactionStateRepoOptions } from '../../../src/infrastructure'
+import { BulkTransactionAgg } from '../../../src/domain'
```

Comment:
Why no alias?

Reply:
The test folder isn't set up to use modules, probably need to add it somewhere in tsconfig.

But mainly I feel very adversed in using the tsconfig alias's given that they result in errors even when there is no discernable circular reference.